kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java

```diff
@@ -162,6 +159,72 @@ public class BigQuerySinkConfig extends AbstractConfig {
       "Whether to automatically sanitize topic names before using them as table names;"
       + " if not enabled topic names will be used directly as table names";
 
+  public static final String TOPIC2TABLE_MAP_CONFIG =                     "topic2TableMap";
+  private static final ConfigDef.Type TOPIC2TABLE_MAP_TYPE =              ConfigDef.Type.STRING;
+  public static final String TOPIC2TABLE_MAP_DEFAULT = "";
+  private static final ConfigDef.Importance TOPIC2TABLE_MAP_IMPORTANCE =  ConfigDef.Importance.LOW;
+  public static final String TOPIC2TABLE_MAP_DOC = "Map of topics to tables (optional). "
+          + "Format: comma-separated tuples, e.g. <topic-1>:<table-1>,<topic-2>:<table-2>,... " +
+          "Note that topic name should not be modified using regex SMT while using this option." +
+          "Also note that if sanitizeTopics is true, then the table names supplied in this config " +
+          "would be sanitized according to the same rules";
+  private static final ConfigDef.Validator TOPIC2TABLE_MAP_VALIDATOR = (name, value) -> {
+    String topic2TableMapString = (String) ConfigDef.parseType(name, value, TOPIC2TABLE_MAP_TYPE);
+
+    if (topic2TableMapString.isEmpty()) {
+      return;
+    }
+
+    Map<String, String> topic2TableMap = new HashMap<>();
+
+    for (String str : topic2TableMapString.split(",")) {
+      String[] tt = str.split(":");
+
+      if (tt.length != 2) {
+        throw new ConfigException(
+                name,
+                topic2TableMapString,
+                "One of the topic to table mappings has an invalid format."
+        );
+      }
+
+      String topic = tt[0].trim();
```

Comment:
while Validation we are considering topic and table name case sensitive. is it true incase of bigquery and kafka topic?

Reply:
Sorry I couldn't find case sensitive check. Can you plz point me to it?